java/arcs/core/storage/ProxyInterface.kt

```diff
@@ -20,7 +20,9 @@ sealed class ProxyMessage<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     /** Identifier for the sender of the [ProxyMessage]. */
     open val id: Int?,
```

Comment:
It looks like this is something that can be easily done in the Kotlin codebase, but not in the TypeScript codebase because of the asynchronous behaviour of registering a storage proxy to a storage endpoint that doesn't exist on the Kotlin side of things.